.github/workflows/release.yml

```diff
@@ -512,6 +512,7 @@ jobs:
         include:
         - arch: x86
         - arch: x64
+        - arch: arm64
```

Comment:
Commit message, please add `[Actions]` in these commits:

`Feature: [Actions] arm64 builds for Windows`

PS: is it ARM64 or arm64? :D

Reply:
Depends if you ask Microsoft or somebody else! Microsoft considers it ARM64.